Mathlib/Algebra/Module/NatInt.lean

```diff
@@ -49,7 +49,7 @@ end AddCommMonoid
 
 section AddCommGroup
 
-variable (R M) [Semiring R] [AddCommGroup M]
+variable [AddCommGroup M]
```

Comment:
This makes `M` implicit: can you say why you think this is better? (At least, this should be mentioned in the PR description - or you can revert this from this PR.)

Reply:
Let's leave this to another PR. (And FWIW, that PR should also say why changing one of these parameters is good. You may know that reason, but it's better if others don't have to mind-read or guess!)